frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -377,7 +377,8 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
       isActivity: (resource) =>
         resource?.status?.phase === 'ImportInProgress' &&
-        !!resource?.metadata?.ownerReferences?.[0],
+        !!resource?.metadata?.ownerReferences?.[0] &&
+        !Number.isNaN(parseInt(resource?.status?.progress, 10)),
```

Comment:
btw, N/A progress might be a valid option for sources where CDI doesn't have progress implemented.

The best solution would be to check importer pod status (if it crashloops) - which is quite a heavy operation though.

Can't we just believe the status of a CRD (treat it as API) and show 0 or `N/A` progress instead?

Reply:
Checking pods is not really an option - as you mentioned - too heavy.
I've opted for showing a generic activity without progress bar if we cannot parse the progress info
![Screenshot from 2020-05-18 14-31-23](https://user-images.githubusercontent.com/2078045/82213594-b4cdc000-9914-11ea-924b-594442d873e7.png)

Reply:
+1